.github/workflows/pkgdown.yaml

```diff
@@ -2,7 +2,7 @@
 # Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    branches: [main, master, config-draft]
+    branches: [main, master, reference-reorg]
```

Comment:
Is a mention of any branch other than `main` still needed after this PR is merged?

Reply:
Good catch!